exampleCourse/questions/workshop/Lesson4_example1/server.py

```diff
@@ -4,6 +4,7 @@
 
 def generate(data):
     A = np.zeros((2, 2))
+    A[0, 1] = 1
```

Comment:
The 'correct answer' isn't actually correct.

Reply:
I don't think this `generate` function is even necessary in the first place since all the grading is done strictly by `grade()` below. Can we just remove it?

Reply:
This is actually useful for determining the correct answer via tests.

Reply:
Can `test()` itself generate the correct answer?

More broadly, as this is a workshop example, I'm far less interested in having the tests work than I am in having a short, easy-to-understand example. If I were in a workshop, I'd ask "why are we specifying a correct answer" and I think any of the possible answers ("I don't know", "Don't worry about it", "Ah! You see, we have this automated system for testing questions...") aren't really anything we want to bring up in a workshop.

Reply:
What about displaying a sample correct answer to the student?

Reply:
Since `test()` runs after the element code, it's way more difficult to do it this way -- now you have to define the raw correct/incorrect/invalid submission for the problem. This is a symptom of how `test()` is designed -- I think that deserves a rethink, but it's not particularly important to me right now.

Reply:
I think generally, I would be excited for a discussion regarding more flexible control flow. E.g. a field you set of where you want the control flow to go. `data["next"] = "test_element"`. Not sure how exactly I would want that to work yet.

Reply:
Fair. I don't want this to hold up this PR; it doesn't make the question any worse than it already is!